force-app/main/default/classes/VariablesDataTypesOperators.cls

```diff
@@ -121,9 +175,17 @@ public with sharing class VariablesDataTypesOperators {
 	 * @return The complete sentence as a single String.
 	 */
 	public static String createSentence(String noun, String verb, String endingPunctuation) {
-		return null; // Replace null with the variable you used to store the result
-	}
-
+		
+        String fullSentence;          
+        if (noun == null || verb == null || endingPunctuation == null)
```

Comment:
I'm not sure I'd use an OR clause here. What if the endingPunctuation is null,  just because of that you'd return null even though the noun and verb are not null.